index.js

```diff
@@ -10,7 +10,7 @@ function isThirteen(n, strictEquality) {
   // plz make way for Count Count.
   if (n === 'https://scontent.cdninstagram.com/hphotos-xtf1/t51.2885-15/s320x320/e35/12237511_444845689040315_1101385461_n.jpg') return true;
 
-  if (n === "Thirteen") return true;
+  if (typeof n === 'string' && n.toLowerCase() === "thirteen".toLowerCase()) return true;
```

Comment:
@Wouter0100 Technically it could be said that thirteen was already lowercased but one must speak to the elegance and clarity of explicitly defining a test case that is not a test case but actually usage but and ever even tried lowercasing other numbers like thirteen toLowerCAse